Tasks/VsTestPlatformToolInstallerV1/vstestplatformtoolinstaller.ts

```diff
@@ -47,6 +52,15 @@ async function startInstaller() {
                 await new NugetFeedInstaller(consolidatedCiData)
                     .getVsTestPlatformToolFromSpecifiedFeed(packageSource, testPlatformVersion, versionSelectorInput, null);
                 break;
+
+            case 'customfeed':
+                await new NugetFeedInstaller(consolidatedCiData)
+                    .getVsTestPlatformToolFromCustomFeed(packageSource, versionSelectorInput, testPlatformVersion, username, password);
```

Comment:
why it's "get" when no value is being returned or someone is not reading on that?

Reply:
okay, will change to install